clients/drcachesim/tests/scheduler_launcher.cpp

```diff
@@ -231,6 +231,21 @@ simulate_core(int ordinal, scheduler_t::stream_t *stream, const scheduler_t &sch
                 thread_sequence += 'A' + static_cast<char>(input % 26);
                 cur_segment_instrs = 0;
             }
+        } else if (record.marker.type == dynamorio::drmemtrace::TRACE_TYPE_MARKER) {
+            if (record.marker.marker_type ==
+                dynamorio::drmemtrace::TRACE_MARKER_TYPE_CPU_ID) {
+#ifdef HAS_ZIP
```

Comment:
nit: the whole else-if block could be conditionally compiled.

Reply:
Done.